src/platform/webengine/WebPageBlink.cpp

```diff
@@ -889,11 +889,14 @@ void WebPageBlink::setPageProperties()
         d->pageView->SetTransparentBackground(true);
     }
 
-    // set inspectable
+#ifdef AGL_DEVEL
+    // Set inspectable. This feature is only available if the 'agl-devel' distro
+    // feature is on.
```

Comment:
IMHO, we could modify here to check also in WebAppManagerConfig if inspector is enabled. If it is not enabled, then we don't ever expose inspector.

Then, in the new implementation of WebAppManagerConfig for AGL we enable inspector only if AGL_DEVEL is set.

Reply:
> IMHO, we could modify here to check also in WebAppManagerConfig if inspector is enabled. If it is not enabled, then we don't ever expose inspector.

As far as I can see, checking in  WebAppManagerConfig if inspector is enabled is already done by the WebPageBlink::inspectable() function.
https://github.com/webosose/wam/pull/21/files/a81fd3ac19f87002aac9ada9ef3e43ef48a74794#diff-60b63322ccf0c48da59259ba2e8b7569R809

So when you say "If it is not enabled, then we don't ever expose inspector." would you like us to change the condition
      if (m_appDesc->isInspectable() || inspectable()) {
to use an AND instead of an OR?.